### ISSUE TITLE
[FEATURE REQUEST] Adding new rows to a table after select via UI

### ISSUE THREAD

**Issue by jonatascb** [sqlectron/sqlectron-gui#441]
Hello, I would like to sugest a new feature for SQLElectron. The ideia is to make it possible to add a couple of rows to a table just by clicking a simple "ADD" button in a result list of a query. After that, SQLElectron should generate the corresponding INSERT for that new added row.

**Comment by barneydmedia** [comment]
I agree that this should be a feature. I would also like to see something similar for adding/editing columns in a table. Hopefully we can make this happen in the future.
